src/status_im/ui/screens/profile/tribute_to_talk/styles.cljs

```diff
@@ -25,10 +25,11 @@
    :line-height 28})
 
 (def description-label
-  {:line-height    22
-   :font-size      15
-   :text-align     :center
-   :color          colors/gray})
+  {:margin-horizontal 16
```

Comment:
Let's remove this margin from here, as it will break other uses of description-label class.